metrics/web/lib/auth/presentation/models/user_profile_model.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+import 'package:metrics/auth/domain/entities/theme_type.dart';
+
+/// A class that represents a user profile model used to
+/// transfer data between [ChangeNotifier]s.
+class UserProfileModel extends Equatable {
```

Comment:
Should we create a `merge` method here as the design document proposes? It should reduce the boilerplate code.